components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/java/org/wso2/carbon/identity/workflow/mgt/ui/WorkflowAdminServiceClient.java

```diff
@@ -160,11 +228,28 @@ public void deleteWorkflow(String workflowId) throws RemoteException, WorkflowAd
         return associations;
     }
 
+    /**
+     * Remove association
+     *
+     * @param associationId  ID of association to remove
+     * @throws RemoteException
+     * @throws WorkflowAdminServiceWorkflowException
+     */
     public void deleteAssociation(String associationId) throws RemoteException, WorkflowAdminServiceWorkflowException {
 
```

Comment:
Usually we keep a blank line after method signature to improve the readability of code, is there any guideline(convention) which ask to prevent using new lines there?